database connection

```diff
@@ -0,0 +1,28 @@
+import torch
```

Comment:
If I remember correctly, Professor Sitao said that we should keep the python code in our repository since we will eventually convert to a python base. In that case, we need to make this a python file and put a comment stating that this code is taken from another repository.